### DISCUSSION TITLE
Context types missing in Nuxt bridge with `@nuxt/types-edge`

### DISCUSSION THREAD

**Discussion by dargmuesli** [vuejs/pinia#1583] [Help and Questions]
Having for example a

```ts
middleware({ $pinia }: Context): void {}
```

gives an `TS2339: Property '$pinia' does not exist on type 'Context'.`
With
- `@nuxt/types-edge: 2.16.0-27616340.013f051b`
- `pinia: 2.0.20`
- `@pinia/nuxt: 0.3.1`

---

I've added a 
```ts
import '@nuxt/types-edge'
import { Pinia } from 'pinia'

declare module '@nuxt/types-edge' {
  interface Context {
    $pinia: Pinia
  }
}
```
to my project as a workaround, but a fix would be nice.

**Comment by nestle49** [comment]
I have the same problem and the same fix works for me

**Reply by dargmuesli** [reply]
You can follow the related issue here: https://github.com/nuxt/bridge/issues/493
